lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -192,19 +192,31 @@ def set_top_view(self):
 
     def _init_axis(self):
         '''Init 3D axes; overrides creation of regular X/Y axes'''
-        self.w_xaxis = axis3d.XAxis('x', self.xy_viewLim.intervalx,
-                                    self.xy_dataLim.intervalx, self)
-        self.xaxis = self.w_xaxis
-        self.w_yaxis = axis3d.YAxis('y', self.xy_viewLim.intervaly,
-                                    self.xy_dataLim.intervaly, self)
-        self.yaxis = self.w_yaxis
-        self.w_zaxis = axis3d.ZAxis('z', self.zz_viewLim.intervalx,
-                                    self.zz_dataLim.intervalx, self)
-        self.zaxis = self.w_zaxis
+        self.xaxis = axis3d.XAxis('x', self.xy_viewLim.intervalx,
+                                  self.xy_dataLim.intervalx, self)
+        self.yaxis = axis3d.YAxis('y', self.xy_viewLim.intervaly,
+                                  self.xy_dataLim.intervaly, self)
+        self.zaxis = axis3d.ZAxis('z', self.zz_viewLim.intervalx,
+                                  self.zz_dataLim.intervalx, self)
 
         for ax in self.xaxis, self.yaxis, self.zaxis:
             ax.init3d()
 
+    @property
+    @cbook.deprecated("3.0", alternative="xaxis")
+    def w_xaxis(self):
+        return self.xaxis
+
+    @property
+    @cbook.deprecated("3.0", alternative="yaxis")
+    def w_yaxis(self):
+        return self.yaxis
+
+    @property
+    @cbook.deprecated("3.0", alternative="zaxis")
```

Comment:
We should give an explicit removal date. These attributes have been here for 10+ years, and I still see them used in submitted examples and questions. So the usual deprecation cycle may not be long enough for most people.

Reply:
+1 to take it slow. It's not hurting much. One could go for a pending deprecation warning first, and maybe target removal="4.0".